### PR TITLE
chore(filters): add the explanation

### PR DESCRIPTION
**Pull request by willmendesneto**

### CODE REVIEW

File: filters/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- [Best Practices](#best-practices)
+- [Doesn't use any special characters and namespace](#doesn-t-use-any-special-characters-and-namespace)
+- [Combining filters](#combining-filters)
+
+
+## Boilerplate
+
+Before start we need to initialize the related filter and mock it's dependencies.
+
+### Boilerplate - Filter file
```
Comment (cironunes) [inline filters/README.md]: We don't need this. You can just remove.

File: filters/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+describe('Filter: trim', function () {
+  var trim;
+
+  beforeEach(module('myApp'));
+
+  beforeEach(inject(function ($filter) {
+    trim = $filter('trim');
+  }));
+
+  it('should return the input string without spaces in left and right sides', function () {
```
Comment (cironunes) [inline filters/README.md]: We don't need the specs here. Can you just remove them?

File: filters/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+# Filters
+> Testing recipes for filters
+
+## Table of contents
+
+- [Boilerplate](#boilerplate)
```
Comment (cironunes) [inline filters/README.md]: We need only one Boilerplate section

File: filters/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+- [Boilerplate](#boilerplate---filter-file)
+- [Boilerplate](#boilerplate---test-for-filter-file)
+- [Best Practices](#best-practices)
```
Comment (cironunes) [inline filters/README.md]: You can remove this section

File: filters/filter.spec.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  it('should return the input string without spaces in left and right sides', function () {
+    var text = '    angularjs  ';
+    expect(trim(text)).toBe('angularjs');
+  });
+
+  it('should return the input string empty if input element value is equal "undefined" or "null" ', function () {
+    expect(trim(undefined)).toBe('');
+    expect(trim(null)).toBe('');
+  });
+
```
Comment (cironunes) [inline filters/filter.spec.js]: unnecessary line

File: filters/filter.spec.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+'use strict';
```
Comment (cironunes) [inline filters/filter.spec.js]: Can you put this ```'use strict';``` statement inside the first ```describe``` block?

File: filters/README.md (hunk 2 above)
Comment (willmendesneto) [inline filters/README.md]: The spec on README.md is based in controller file, that already have too.

**Comment by cironunes** [comment]
Reviewed, fixed and landed.

Thanks!

**Pull request closed without merging**